### PR TITLE
Added Client IP authentication

### PR DESCRIPTION
**Pull request by cyberblob**
This allows you to restrict RestPS to only be accessed from specific IP's

### CODE REVIEW

**Comment by jpsider** [comment]
Looks like there are still a couple bud failures. Are you able to see them? Want to tackle them or for me to take a crack?

**Comment by cyberblob** [comment]
All cleared up now.

File: RestPS/private/Invoke-ValidateIP.ps1 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        This will return a boolean.
+    #>
+    [CmdletBinding()]
+    [OutputType([boolean])]
+    param(
+        [Parameter()][Bool]$VerifyClientIP
+    )
+    if ($VerifyClientIP -eq $true)
+    {
+        . c:\restps\bin\Get-RestIPAuth.ps1
```
Comment (jpsider) [inline RestPS/private/Invoke-ValidateIP.ps1:20]: Not a huge fan of hard-coding paths, this would not work on Linux.

**Comment by cyberblob** [comment]
Oh ya we can fix that. It does not need to be hard coded just need to be in bin. Need to add back the param I bet then fix the path. No prob

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by jpsider** [comment]
RestPS 7.0.42 - was just published to the PowerShell Gallery.

**Pull request merged**